modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -148,37 +148,45 @@ export default {
       const verb = this.relationshipField ? 'Choose' : 'Manage';
       return `${verb} ${this.moduleLabels.pluralLabel}`;
     },
-    options() {
+    moduleOptions() {
       return window.apos.modules[this.moduleName];
     },
     toolbarFilters() {
-      if (!this.options || !this.options.filters) {
+      if (!this.moduleOptions || !this.moduleOptions.filters) {
         return null;
       }
 
-      return this.options.filters.filter(filter => {
+      return this.moduleOptions.filters.filter(filter => {
         // Removes _tags since that will be in the left sidebar.
         return filter.name !== '_tags';
       });
     },
     moduleLabels() {
-      if (!this.options) {
+      if (!this.moduleOptions) {
         return null;
       }
       return {
-        label: this.options.label,
-        pluralLabel: this.options.pluralLabel
+        label: this.moduleOptions.label,
+        pluralLabel: this.moduleOptions.pluralLabel
       };
     },
     selected() {
       return this.items.filter(item => this.checked.includes(item._id));
     },
     accept() {
-      return this.options.schema.find(field => field.name === 'attachment').accept;
+      return this.moduleOptions.schema.find(field => field.name === 'attachment').accept;
     },
     // Whether a cancellation requires confirmation or not
     isModified () {
       return (this.editing && this.modified) || this.relationshipIsModified();
+    },
+    headers() {
```

Comment:
There's a confusing loop here (edit: actually less of a loop than I thought originally).

The media manager doesn't have table headers or columns. It looks like this is only used in `AposDocsManagerMixin` for `generateIcons`, but the media manager also doesn't use `icons`. Why do we need to do anything special other than return an empty array?